lib/mayaUsd/ufe/UsdTransform3dMayaXformStack.cpp

```diff
@@ -169,6 +173,72 @@ createTransform3d(const Ufe::SceneItem::Ptr& item, NextTransform3dFn nextTransfo
     return stackOps.empty() ? nextTransform3dFn() : UsdTransform3dMayaXformStack::create(usdItem);
 }
 
+// Class for setMatrixCmd() implementation.  Should be rolled into a future
+// command class with full undo / redo support
```

Comment:
This temporary command just does undo / redo with values.  Will be rolled into the up-coming Uber value undo / redo class.